testing/kuttl/e2e/cluster-migrate/00--non-crunchy-cluster.yaml

```diff
@@ -0,0 +1,195 @@
+type: Opaque
+data:
+  postgres-password: "U1I2a05BRlh2WA=="
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: non-crunchy-cluster-hl
+  labels:
+    postgres-operator-test: kuttl
+    app.kubernetes.io/name: postgresql
+    app.kubernetes.io/instance: non-crunchy-cluster
+    app.kubernetes.io/component: primary
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
+spec:
+  type: ClusterIP
+  clusterIP: None
+  publishNotReadyAddresses: true
+  ports:
+    - name: tcp-postgresql
+      port: 5432
+      targetPort: tcp-postgresql
+  selector:
+    app.kubernetes.io/name: postgresql
+    app.kubernetes.io/instance: non-crunchy-cluster
+    app.kubernetes.io/component: primary
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: non-crunchy-cluster
+  labels:
+    postgres-operator-test: kuttl
+    app.kubernetes.io/name: postgresql
+    app.kubernetes.io/instance: non-crunchy-cluster
+    app.kubernetes.io/component: primary
+spec:
+  type: ClusterIP
+  sessionAffinity: None
+  ports:
+    - name: tcp-postgresql
+      port: 5432
+      targetPort: tcp-postgresql
+      nodePort: null
+  selector:
+    app.kubernetes.io/name: postgresql
+    app.kubernetes.io/instance: non-crunchy-cluster
+    app.kubernetes.io/component: primary
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: non-crunchy-cluster
+  labels:
+    postgres-operator-test: kuttl
+    app.kubernetes.io/name: postgresql
+    app.kubernetes.io/instance: non-crunchy-cluster
+    app.kubernetes.io/component: primary
+spec:
+  replicas: 1
+  serviceName: non-crunchy-cluster-hl
+  updateStrategy:
+    rollingUpdate: {}
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      postgres-operator-test: kuttl
+      app.kubernetes.io/name: postgresql
+      app.kubernetes.io/instance: non-crunchy-cluster
+      app.kubernetes.io/component: primary
+  template:
+    metadata:
+      name: non-crunchy-cluster
+      labels:
+        postgres-operator-test: kuttl
+        app.kubernetes.io/name: postgresql
+        app.kubernetes.io/instance: non-crunchy-cluster
+        app.kubernetes.io/component: primary
+    spec:
+      serviceAccountName: default
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - podAffinityTerm:
+                labelSelector:
+                  matchLabels:
+                    postgres-operator-test: kuttl
+                    app.kubernetes.io/name: postgresql
+                    app.kubernetes.io/instance: non-crunchy-cluster
+                    app.kubernetes.io/component: primary
+                namespaces:
+                  - "default"
+                topologyKey: kubernetes.io/hostname
+              weight: 1
+      securityContext:
+        fsGroup: 1001
+      hostNetwork: false
+      hostIPC: false
+      containers:
+        - name: postgresql
+          image: docker.io/bitnami/postgresql:14.5.0-debian-11-r37
```

Comment:
If we want to run this in our Jenkins, with envsubst for different versions, we're going to have to figure out some way to parameterize this...

Reply:
(Solved in the Makefile for now.)